### PR TITLE
[Tests] Increase file descriptor limit

### PR DESCRIPTION
**Pull request by bendikro**

### CODE REVIEW

**Comment by cas--** [comment]
I think `openbt.png` can be deleted, I removed the test but forgot about the icon.

**Comment by bendikro** [comment]
Removed

File: deluge/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Increase open file descriptor limit to allow tests to run
+# without getting error: what():  epoll: Too many open files
+import resource
+
+try:
+    resource.setrlimit(resource.RLIMIT_NOFILE, (65536, 65536))
```
Comment (cas--) [inline deluge/tests/__init__.py]: Does Windows fail with ValueError for this?

File: deluge/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Increase open file descriptor limit to allow tests to run
+# without getting error: what():  epoll: Too many open files
+import resource
```
Comment (cas--) [inline deluge/tests/__init__.py]: This will `ImportError` on Windows

**Comment by bendikro** [comment]
Does this look better?

File: deluge/tests/__init__.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+try:
+    resource.setrlimit(resource.RLIMIT_NOFILE, (65536, 65536))
+except ValueError as ex:
+    print "Failed to raise file descriptor limit:", ex
```
Comment (aresch) [inline deluge/tests/__init__.py]: Please use the print function from __future__: from __future__ import print_function

Comment (bendikro) [inline deluge/tests/__init__.py]: Sure

**Comment by cas--** [comment]
Merged, thanks!

**Pull request closed without merging**